yaml/pois.yaml

```diff
@@ -316,8 +316,10 @@ filters:
     output:
       kind: station
       state: {col: tags->state}
-  - filter:
-      railway: [halt, tram_stop, stop]
+  - filter: 
+      all:
```

Comment:
Don't need this `all` here, it's implied, but you can keep it if you think it makes it more readable.